awslambda/entrypoint.py

```diff
@@ -38,6 +40,11 @@ def lambda_handler(event: Dict[str, Any], context) -> Dict[str, Any]:
             pcluster_manager_api = _init_flask_app()
         # Setting default region to region where lambda function is executed
         os.environ["AWS_DEFAULT_REGION"] = os.environ["AWS_REGION"]
+
+        if 'requestContext' in event and 'requestId' in event['requestContext']:
+            # set apigw requestId to allow Flask app context to use it in logging
+            set_apigw_request_id_global(event["requestContext"]["requestId"])
```

Comment:
question: the `context` is already forwarded to the application with https://github.com/aws-samples/pcluster-manager/blob/cffc31d946abe90dd87683441ee9d37290d5b588/awslambda/serverless_wsgi.py#L263 can we re-use this part to access the `requestId` and get rid of the `LocalProxy`?
Probably an `os.environ('serverless.context')` is sufficient

Reply:
I'll try again, I looked into it but wasn't able to

Reply:
changed in next rev